### PR TITLE
Use Play Widget

### PR DESCRIPTION
**Pull request by SylvainCorlay**
This depends on https://github.com/ipython/ipywidgets/pull/633 which adds a front-end only `Play` widget that uses the event loop of the browser.

Will require a release of ipywidgets before merging.

won

### CODE REVIEW

File: examples/Wealth of Nations.ipynb [hunk 1]
```diff
@@ -196,8 +196,8 @@
    },
    "outputs": [],
    "source": [
-    "ax_y = Axis(label='Life Expectancy', scale=y_sc, orientation='vertical', side='left')\n",
-    "ax_x = Axis(label='Income per Capita', scale=x_sc)"
+    "ax_y = Axis(label='Life Expectancy', scale=y_sc, orientation='vertical', side='left', grid_lines='solid')\n",
```
Comment (dmadeka) [inline examples/Wealth of Nations.ipynb]: Not sure about this, unless you make the grid lines super light. The line won't show, and in this case, I guess you dont care too much about seeing the exact value, more about seeing the relative positions and the paths?

**Comment by SylvainCorlay** [comment]
Rebased

File: examples/Wealth of Nations.ipynb (hunk 1 above)
Comment (dmadeka) [inline examples/Wealth of Nations.ipynb]: Lets see a screen shot?

**Comment by SylvainCorlay** [comment]
This is displaying the new style too in the screenshot.

**Comment by dmadeka** [comment]
@SylvainCorlay Sorry which screenshot?

**Comment by SylvainCorlay** [comment]
now it is there :)

**Comment by dmadeka** [comment]
Okay, beautiful. Im good to merge this.

**Comment by SylvainCorlay** [comment]
ok, I also updated made an ipywidget 5.2.0 release and updated this setup.py.

**Comment by SylvainCorlay** [comment]
:shipit:

**Comment by SylvainCorlay** [comment]
:shipit: !

**Comment by dmadeka** [comment]
Sorry for the delay! There you go

**Comment by dmadeka** [comment]
@SylvainCorlay  bqplot 0.7.0?

**Comment by SylvainCorlay** [comment]
Yes, I am doing the ipywidgets conda-forge release first.

We could do bqplot tonight or tomorrow, but I guess we should talk first!

**Comment by dmadeka** [comment]
We'll all call you at 11am tomorrow?

@rmenegaux @ssunkara1

**Pull request merged**